src/ledger.rs

```diff
@@ -553,13 +553,14 @@ pub fn tmp_ledger_path(name: &str) -> String {
 }
 
 #[cfg(test)]
-pub fn genesis(name: &str, num: i64) -> (Mint, String) {
+pub fn genesis(name: &str, num: i64) -> (Mint, String, Vec<Entry>) {
     let mint = Mint::new(num);
     let path = tmp_ledger_path(name);
     let mut writer = LedgerWriter::open(&path, true).unwrap();
-    writer.write_entries(mint.create_entries()).unwrap();
+    let entries = mint.create_entries();
+    writer.write_entries(entries.clone()).unwrap();
 
-    (mint, path)
+    (mint, path, entries)
```

Comment:
What's the point of this change? Why wouldn't the caller call `mint.create_entries()`?

Reply:
You're right, I didn't realize that create_entries() would create the same entries every time it's called xD